lib/rubocop/cop/rails/file_path.rb

```diff
@@ -180,6 +180,11 @@ def autocorrect_file_join(corrector, node)
               side: :right
             )
           )
+          node.arguments.each do |argument|
+            next unless argument.str_type?
+
+            corrector.replace(argument, argument.value.delete_prefix('/').inspect)
+          end
```

Comment:
CI is failed: https://circleci.com/gh/rubocop/rubocop-rails/17239
Can you tweak file_path.rb to the following?

```suggestion
          node.arguments.filter(&:str_type?).each do |argument|
            corrector.replace(argument, argument.value.delete_prefix('/').inspect)
          end
```

Reply:
Oh, sorry. The local rubocop was old and I didn't notice. I updated this PR.